src/main.cpp

```diff
@@ -1,7 +1,7 @@
 /* Include files */
 /* System Includes*/
 #include <Arduino.h>
-#include "ParameterInterface.h"
+// #include "ParameterInterface.h"
```

Comment:
Why is ParameterInterface not needed in main anymore?

Reply:
it is no longer a thing that exists. the parameter interface may exist in the future, however it is something that doesnt currently exist nor is planned to in the near future.